generate_sysnames.py

```diff
@@ -117,9 +117,10 @@ def __init__(self, version, **kwargs):
 # Update this when a new release comes online
 # add "deprecated=True" to distros when we stop building for
 # them.
-distros = [Ubuntu('18.04'),
-           Ubuntu('17.10'),
-           Ubuntu('17.04'),
+distros = [Debian('10.0'),
+           Ubuntu('18.04'),
```

Comment:
Ubuntu 18.04 also has base-files 10. We should check that this doesn’t switch Ubuntu 18.04’s primary sysname to deb10, and think about whether we want Ubuntu 18.04 to have deb10 at all.

Reply:
Good catch. This seems to in fact cause an issue:

$ OVERRIDE_MACHTYPE_LSB_ID=ubuntu OVERRIDE_MACHTYPE_DEB_VERSION=10.1ubuntu2.3 OVERRIDE_MACHTYPE_UBUNTU_VERSION=18.04 generate_sysnames.py 
Using values: Debian: 10.1ubuntu2.3, Ubuntu: 18.04, Arch: amd64
generate_sysnames.py: Sysname mismatch -- is this a new release?
generate_sysnames.py: amd64_ubuntu1804 != amd64_deb100

buster released a whole year after Bionic, so I'm leaning against having Bionic grow a deb100 sysname.

I've commited a fix.